src/lib/layout/navbar/index.tsx

```diff
@@ -36,49 +39,25 @@ const Navbar = observer(({ isExpand, setIsExpand }: NavbarProps) => {
   const isCurrentPage = useIsCurrentPage();
   const wasm = useWasmConfig({ shouldRedirect: false });
   const move = useMoveConfig({ shouldRedirect: false });
-
+  const tier = useTierConfig({ minTier: "lite" });
   const { address } = useCurrentChain();
 
   const navMenu: MenuInfo[] = [
     {
       category: "Your Account",
       slug: "your-account",
       submenu: [
-        {
-          name: "Past Transactions",
-          slug: "/past-txs",
-          icon: "history" as IconKeys,
-        },
-        {
-          name: "Your Account Details",
-          slug: `/accounts/${address}`,
-          icon: "admin" as IconKeys,
-          isDisable: !address,
-          tooltipText:
-            "You need to connect wallet to view your account details.",
-          trackEvent: () => track(AmpEvent.USE_TO_YOUR_ACCOUNT),
-        },
+        ...getYourAccountSubmenu(address),
+        ...(tier === "lite"
```

Comment:
I suggest creating a separate file for lite tier insteads of adding condition here.